src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--fri calendar--month-length--28">
+      <div class="calendar__day day-1"></div>
```

Comment:
leave just calendar__day class on each day

```suggestion
      <div class="calendar__day"></div>
```

Reply:
I need these classes to add the dates to calendar : 
@for $index from 1 through 31 {
  .day-#{$index}::before {
    content: "#{$index}";
    font-family: Arial, Helvetica, sans-serif;
    font-size: 30px;
  }
}

Reply:
use nth child